src/jit/lsra.h

```diff
@@ -921,6 +921,11 @@ class LinearScan : public LinearScanInterface
         assignPhysReg(getRegisterRecord(reg), interval);
     }
 
+#ifdef _TARGET_ARM_
+    bool isAssigned(RegRecord* regRec, RegisterType newRegType);
+    bool isUnassigned(RegRecord* regRec, LsraLocation lastLocation, RegisterType newRegType);
+#endif
+    bool isUnassigned(RegRecord* regRec, LsraLocation lastLocation);
```

Comment:
I think all of these should be changed to `isAssigned` for consistency (and to avoid the double negation of `!isUnassigned()`.